pkg/detectors/gitlab/v1/gitlab.go

```diff
@@ -60,15 +66,21 @@ func (s Scanner) FromData(ctx context.Context, verify bool, data []byte) (result
 		s1 := detectors.Result{
 			DetectorType: detectorspb.DetectorType_Gitlab,
 			Raw:          []byte(resMatch),
+			ExtraData:    map[string]string{},
 		}
 		s1.ExtraData = map[string]string{
 			"rotation_guide": "https://howtorotate.com/docs/tutorials/gitlab/",
 			"version":        fmt.Sprintf("%d", s.Version()),
 		}
 
 		if verify {
-			isVerified, verificationErr := s.verifyGitlab(ctx, resMatch)
+			isVerified, isUserBlocked, verificationErr := s.verifyGitlab(ctx, resMatch)
 			s1.Verified = isVerified
+			// if user account is blocked, add this information in the extra data
+			if isUserBlocked {
+				s1.ExtraData["User Account Blocked"] = "Yes"
```

Comment:
This is a bit wordy. I think just "blocked = true" would be fine. 
https://docs.gitlab.com/ee/api/users.html#as-a-regular-user

e.g., https://github.com/trufflesecurity/trufflehog/blob/871a2b0f06f602899f8c928ce19d2737a3758e6c/pkg/detectors/github/v1/github_old.go#L152-L154